chapters/mitigations-and-defensive-strategies/defense-mechanisms/activities/04-rwslotmachine4/sol/README.md

```diff
@@ -0,0 +1,11 @@
+# Building and Running
+
+1. Build the Docker image:
```

Comment:
Add empty line before triple backticks.

Indent snippet to rest below `B` (from `Build`).